core/platform/datastore/cloud_datastore_services.py

```diff
@@ -47,12 +47,15 @@
 
 
 def get_ndb_context(**kwargs):
-    """Get the context of the Cloud NDB. This context need to be entered in
+    """Get the context of the Cloud NDB. This context needs to be entered in
     order to do any Cloud NDB operations.
 
     Returns:
         ndb.context.Context. Cloud NDB context.
     """
+    # The raise_context_error is set to False, because we want to generate
```

Comment:
raise_context_error --> raise_context_error arg
new context --> a new context

That said, I'm not sure I follow. Wouldn't None just be returned in such a case? I don't see anything in the impl about creating a new context. (The returns docstring should also be updated to include None as a type if a return value of None is a possibility.)

Reply:
Few lines below we create the new context `CLIENT.context(**kwargs) if context is None`

Reply:
Oh weird, maybe I was looking at an old version. Thanks for clarifying.